### PR TITLE
Show real stats on home page

### LINKED ISSUE #378: Use stats from UserImpressions on the home page
Related to #342

### PR DESCRIPTION
**Pull request by hopsoft**
Fixes #378 

codefund 2018-10-10 17-10-56

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,4 +31,5 @@ cover
 *.mmdb
 .DS_Store
 *.tar.gz
-.vscode
\ No newline at end of file
+.vscode
+teamocil
```
Comment (StareIntoTheBeard) [inline .gitignore]: what's that

File: lib/code_fund/stats/user_impressions.ex [hunk 2]
```diff
@@ -50,62 +53,51 @@ defmodule CodeFund.Stats.UserImpressions do
...
     click_count = click_count_for_last_thirty_days()
+    paid_click_count = paid_click_count_for_last_thirty_days()
 
     click_rate =
       case click_count do
         0 -> 0.0
         _ -> click_count / impression_count
       end
 
+    paid_click_rate =
```
Comment (StareIntoTheBeard) [inline lib/code_fund/stats/user_impressions.ex]: Can this logic be pulled into a private function?

File: lib/code_fund/queries/user_impression.ex [hunk 3]
```diff
@@ -0,0 +1,70 @@
+defmodule CodeFund.Query.UserImpression do
```
Comment (StareIntoTheBeard) [inline lib/code_fund/queries/user_impression.ex]: Does any of this need testing or are we satisfied with the tests in the Stats module?

File: .gitignore (hunk 1 above)
Comment (hopsoft) [reply .gitignore]: It's what I use to manage my local tmux environment for the project. 

https://github.com/remiprev/teamocil

It's pretty nice. From the root of the project I simply run

```sh
bin/teamocil nate
```

...and presto I have a tmux session with several windows/panes running for my dev environment. Each has the correct version of Erlang & Elixir & sources `.env` etc...

- vim
- repl
- phoenix server
- console
- etc...

I've actually committed my teamocil configs on other repos as it helped other team members learn about the app environment & get setup themselves (even when not using tmux/teamocil).

_Not sure it's appropriate to commit on this project though._

File: lib/code_fund/queries/user_impression.ex (hunk 3 above)
Comment (hopsoft) [reply lib/code_fund/queries/user_impression.ex]: I'd like to get some tests on it.

File: .gitignore (hunk 1 above)
Comment (StareIntoTheBeard) [reply .gitignore]: Might suggest you throw that in your local global gitignore in that case.

Comment (hopsoft) [reply .gitignore]: Considered it but I have about 7 other projects that have it committed.

Comment (StareIntoTheBeard) [reply .gitignore]: Sure, but does everyone use that tool? I don't; its a good case for your local global gitignore if its something you use on every project.

**Pull request merged**